### DISCUSSION TITLE
SQLite: Foreign key constraints are not enforced

### DISCUSSION THREAD

**Discussion by jjkavalam** [quarkusio/quarkus#29362] [Q&A]
Despite declaring `foreign key` constraint in my `create table` statement, foreign key constraints are not enforced.

I read elsewhere that it requires:
- an SQLite version that is compiled with foreign key constraints enabled
- each connection needs to be configured

Is there a quick and easy way to turn on foreign key constraints when using SQLite with Quarkus / Hibernate ?

**Comment by ranjanashish** [comment]
The following works
```
quarkus.datasource.jdbc.url=jdbc:sqlite:<path-to-file>?foreign_keys=ON
```
Tested with Quarkus 3.9.3. This should work with older and future versions too.
